packages/dso-toolkit/src/styles/components/_tabs.scss

```diff
@@ -24,7 +24,6 @@ ul.nav-tabs {
 
   > li {
     border-color: $dso-tabs-color;
-    flex: 1;
```

Comment:
Is dan in de `@media screen and (max-width: $screen-xs-max)` nog wel `flex: 0;` nodig, of kan die ook weg...?

Reply:
die kan ook weg, will fix